docs/reference/release-notes/6.3.asciidoc

```diff
@@ -1,3 +1,55 @@
+[[release-notes-6.3.2]]
+== 6.3.2 Release Notes
+
+Also see <<breaking-changes-6.3>>.
+
+[[bug-6.3.2]]
+[float]
+=== Bug fixes
+
+Allocation::
+* A replica can be promoted and started in one cluster state update {pull}32042[#32042]
+
+Authentication::
+* Fix building AD URL from domain name {pull}31849[#31849]
+
+Core::
+* Disable C2 from using AVX-512 on JDK 10 {pull}32138[#32138] (issue: {issue}31425[#31425])
+
+Machine Learning::
+* Fix calendar and filter updates from non-master nodes {pull}31804[#31804] (issue: {issue}31803[#31803])
+* Don't treat stale FAILED jobs as OPENING in job allocation {pull}31800[#31800] (issue: {issue}31794[#31794])
+* Rate limit established model memory updates {pull}31768[#31768]
+* Wait for aliases in multi-node tests {pull}32086[#32086] (issue: {issue}32034[#32034])
+
+Mapping::
+* Fix put mappings java API documentation {pull}31955[#31955] (issue: {issue}31906[#31906])
```

Comment:
I think we should remove this: it's a docs change, but the PR was not labelled as such. I've pinged @cbuescher for a second opinion, but I think we should remove this and label #31955 with `>docs`